### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by biswabaibhab007**
Added a ReactJS course in hindi by 6 Pack Programmer.

## What does this PR do?
Adds a youtube playlist for ReactJS in hindi.

### Description
A playlist for ReactJS in hindi.

### Why is this valuable (or not)?
The tutor uses daily life examples for better understanding.

### How do we know it's really free?
It's on youtube.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a video course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
Fix the linter error.

**Comment by biswabaibhab007** [comment]
> Fix the linter error.

@Ajay-singh1 can you help me with it?
I didn't quite understand what I did wrong.

**Comment by Ajay-singh1** [comment]
@biswabaibhab007 Check if the Indentation is in proper format and the list is in alphabetical order also read Contributing.md file to be clear.

**Comment by biswabaibhab007** [comment]
@Ajay-singh1, Yes the indentation as well as the alphabetical sequence is correct

**Comment by biswabaibhab007** [comment]
There was an error in the alphabetical sequence. I fixed it and raised a new PR because I don't know how to make changes to the current PR.

**Comment by eshellman** [comment]
@biswabaibhab007 we will not review the new PR, you need to learn how to make changes to your PR.

**Comment by biswabaibhab007** [comment]
> @biswabaibhab007 we will not review the new PR, you need to learn how to make changes to your PR.

@eshellman It'd be  really helpful if you can suggest me some article

**Comment by biswabaibhab007** [comment]
Thanks @eshellman learnt something new because of you. 😇 
This PR can now be merged.

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -431,6 +431,7 @@
 #### React
 
 * [React JS (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhignjLLXTJWkRJKN-SgAqClL) - Rajesh Kumar, Geeky Shows
+* [React JS COURSE 2023](https://youtube.com/playlist?list=PLt5mNkGuWcuWSUHxSzWP74IU9U4BTVLt0&si=Z_vofUlaK3rVj_UX) - 6 Pack Programmer
```
Comment (eshellman) [inline courses/free-courses-hi.md:434]:
```suggestion
* [React JS COURSE 2023](https://youtube.com/playlist?list=PLt5mNkGuWcuWSUHxSzWP74IU9U4BTVLt0) - 6 Pack Programmer
```
remove tracking parameter.
Thanks for keeping the same PR!

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -431,6 +431,7 @@
 #### React
 
 * [React JS (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhignjLLXTJWkRJKN-SgAqClL) - Rajesh Kumar, Geeky Shows
+* [React JS COURSE 2023](https://youtube.com/playlist?list=PLt5mNkGuWcuWSUHxSzWP74IU9U4BTVLt0) - 6 Pack Programmer
```
Comment (eshellman) [inline courses/free-courses-hi.md:434]:
```suggestion
* [React JS Course 2023](https://youtube.com/playlist?list=PLt5mNkGuWcuWSUHxSzWP74IU9U4BTVLt0) - 6 Pack Programmer
```
No ALL CAPS

**Pull request merged**